builders/get-started/eth-compare/consensus-finality.md

```diff
@@ -13,48 +13,56 @@ While Moonbeam strives to be compatible with Ethereum's Web3 API and EVM, there
 
-In terms of consensus, Moonbeam is based on Delegated Proof-of-Stake, while Ethereum relies on Proof-of-Work, which are very different. Consequently, Proof of Work concepts, such as  `difficulty`, `uncles`, `hashrate`, generally don’t have meaning within Moonbeam.
+In terms of consensus, Moonbeam is based on Delegated Proof-of-Stake, while Ethereum relies on a standard Proof-of-Stake system, which is slightly different. Although both mechanisms rely on the use of stake to validate and create new blocks, there are some key differences.
 
-For APIs that return values related to Ethereum’s Proof of Work, default values are returned. Existing Ethereum contracts that rely on Proof of Work internals (e.g., mining pool contracts) will almost certainly not work as expected on Moonbeam.
+With PoS on Ethereum, validators are selected to produce and validate blocks based on their own stake in the network. As long as a validator has placed a validator deposit, they can be selected to produce and validate blocks. However, as previously mentioned, the greater the stake in the network, the higher the chances a validator has to be selected to produce and validate blocks.
 
-However, the deterministic finality of Moonbeam can be used to provide a better user experience than is currently possible in Ethereum. The strategy to check for transaction finality is fairly simple:
+On the other hand, with DPoS on Moonbeam, collators are selected to produce blocks based on their own stake plus their delegated stake in the network. Any token holder can choose to delegate their stake to a collator candidate. However, not all collator candidates are eligible to produce and validate blocks, they must be in the active set. The active set consists of the top candidates by stake, including delegations. The number of candidates in the active set is subject to [governance](/learn/features/governance){target=_blank}.
+
+In terms of finality, blocks on Ethereum can take quite a bit longer to finalize than on Moonbeam due to the checkpoint finality system it uses. In Ethereum, validators determine finality at checkpoint blocks, which are always the first block in an epoch. Since an epoch has 32 slots and each slot is 12 seconds, it'll take at least 384 seconds, or 6.4 minutes for a block to be finalized.
+
+Moonbeam does not use checkpoint blocks and instead relies on the GRANDPA finality gadget, where the finality process is completed in parallel to block production. In addition, the finality process incorporates the blockchain's structure, which allows the relay chain validators to vote on the highest block that they think is valid. In this scenario, the vote would apply to all of the blocks leading up to the one that is finalized, which speeds up the finalization process. After a block has been included in the relay chain, a block can be finalized within one block on Moonbeam.
+
+## Strategy to Check Transaction Finality {: #strategy-to-check-tx-finality }
```

Comment:
Ethereum introduced some new block tags like `finalized` etc as part of the PoS merge. We now support the same block tags in the Frontier RPC, so maybe there is another strategy here that we can show? Ethers.js and other libraries might support the block tags